src/main/sql/Database Scripts.sql

```diff
@@ -1,5 +1,8 @@
 CREATE DATABASE TEAMMANAGEMENTSYSTEM;
```

Comment:
As discussed please make the name short and sweet when possible - like TMS. In case you wish to have a longer name for any reason, you a separator character for a better readability like TEAM_MGMT_SYSTEM etc.,

Reply:
Thank for the suggestion, I will work on it

Reply:
Comments are incorporated, please do a further review. thanks